dynamixels:
  port_info:
    port_name: /dev/ttyACM0
    baudrate: 2000000
    protocol_version: 2
  control_mode: position
  auto_torque: true
  device_info:
    HeadPan:
      id: 1
      model_number: 321
    HeadTilt:
      id: 2
      model_number: 311
    LShoulderPitch:
      id: 3
      model_number: 311
    LShoulderRoll:
      id: 4
      model_number: 311
    LElbow:
      id: 5
      model_number: 311
    RShoulderPitch:
      id: 6
      model_number: 311
    RShoulderRoll:
      id: 7
      model_number: 311
    RElbow:
      id: 8
      model_number: 311
    LHipYaw:
      id: 9
      model_number: 311
    LHipRoll:
      id: 10
      model_number: 311
    LHipPitch:
      id: 11
      model_number: 311
    LKnee:
      id: 12
      model_number: 311
    LAnklePitch:
      id: 13
      model_number: 311
    LAnkleRoll:
      id: 14
      model_number: 311
    RHipYaw:
      id: 15
      model_number: 311
    RHipRoll:
      id: 16
      model_number: 311
    RHipPitch:
      id: 17
      model_number: 311
    RKnee:
      id: 18
      model_number: 311
    RAnklePitch:
      id: 19
      model_number: 311
    RAnkleRoll:
      id: 20
      model_number: 321

  read_values:
    read_position: true
    read_velocity: true
    read_effort: true
IMU:
  name: IMU
  frame: IMU_frame
  read: true
      
  